src/components/ContactForm.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from "react";
+import {
+  TextField,
+  Box,
+  Button,
+  Typography,
+  Select,
+  FormControl,
+  InputLabel,
+} from "@material-ui/core";
+import Inquiries from "../modules/Inquiries";
+import { useSelector } from "react-redux";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  formContainer: {
+    [theme.breakpoints.up("xs")]: {
+      padding: "0",
+    },
+    [theme.breakpoints.up("md")]: {
+      padding: "0 50px",
+    },
+  },
+}));
+
+const RentOutForm = () => {
```

Comment:
As much as I love code reuse, copying code straight up from a project will not fly.

Reply:
Fixed